src/main/kotlin/state/State.kt

```diff
@@ -0,0 +1,12 @@
+package state
+
+import Hand
+import card.PlayingCard
+
+interface State {
```

Comment:
You've finally applied the state pattern to this mission! 👍👍
Great to see you reach your goal!